jobbergate-composed/docker-compose.yml

```diff
@@ -156,7 +156,7 @@ services:
       retries: 5
 
   minio:
-    image: minio/minio
+    image: minio/minio:RELEASE.2023-10-25T06-33-25Z
```

Comment:
Task created to investigate and revert this change when possible: https://app.clickup.com/t/18022949/PENG-1829